cmd/otel-allocator/main.go

```diff
@@ -47,7 +48,9 @@ func main() {
 	ctx := context.Background()
 
 	log := ctrl.Log.WithName("allocator")
-	allocator := allocation.NewAllocator(log)
+
+	strategy, _ := allocation.NewStrategy(*cliConf.AllocationStrategy)
```

Comment:
Catch this error here.  The application can't start if there is no valid allocation strategy.

Reply:
yep, this was because of a refactor by intellij